### PR TITLE
Update running-a-full-node.md

### PR DESCRIPTION
**Pull request by KoenBal**
First PR any remarks about the process (wrong branch or any other remarks) welcome.

### Description
Step numbering was not really clear. I assumed configuring the node...
More detailed phrasing.

### CODE REVIEW

File: packages/docs/getting-started/running-a-full-node.md [hunk 1]
```diff
@@ -40,15 +40,15 @@ $ cd celo-data-dir
 
 ## **Create an account and get its address**
 
-In this step, you'll create an account on the network. If you've already done this and have an account address, you can skip this and move on to step 3.
+In this step, you'll create an account on the network. If you've already done this and have an account address, you can skip this and move on to configuring the node.
```
Comment (koenprivate) [inline packages/docs/getting-started/running-a-full-node.md:43]: Less confusing

**Comment by KoenBal** [comment]
Does the CI need to run when updating only .md files or the docs?

**Comment by annakaz** [comment]
Thanks for the contribution, KoenBal! Looks like these changes to the readme already got merged in, so I'll go ahead and close this PR.

**Comment by KoenBal** [comment]
@annakaz Thx for reviewing. Maybe one last question :). How can you see they were already merged? Shouldn't the changes be visible in the docs then? Note that the PR is against the alfajores docs (I used the "Edit on GitHub" button on the docs page BTW) Anyways I assume it doesn't really matter anymore as only the master docs should be used from now on? Should the option to go to alfajores docs (left top button on docs) still be there?

**Pull request closed without merging**